packages/wrangler/src/api/integrations/bindings/index.ts

```diff
@@ -0,0 +1,151 @@
+import { Miniflare } from "miniflare";
+import { readConfig } from "../../../config";
+import { getBindings } from "../../../dev";
+import { getBoundRegisteredWorkers } from "../../../dev-registry";
+import { buildMiniflareBindingOptions } from "../../../dev/miniflare";
+import { getServiceBindings } from "./services";
+import type { MiniflareOptions } from "miniflare";
+
+/**
+ * Options for the `getBindingsProxy` utility
+ */
+export type GetBindingsProxyOptions = {
```

Comment:
So we are not allowing the developer to manually specify what bindings they would like to proxy to?
They have to be defined in a config file?

Reply:
Is this to avoid having to commit to the shape of that config?